src/app.js

```diff
@@ -8,6 +8,7 @@ let reportsRouter = require('./reports/routes/reportsRoute.js');
 let configRouter = require('./configManager/routes/configRoute.js');
 let dslRouter = require('./tests/routes/dslRoute.js');
 let testsRouter = require('./tests/routes/testsRoute.js');
+const processorsFilesRouter = require('./processorsFiles/routes/processorsFileRoutes');
```

Comment:
I would change all of the `processorsFiles*` to `processors*` in the file names and code, since this isn't necessarily a file - if type is `javascript` for example

Reply:
I agree, will resolve this asap

Reply:
I have pushed the updated variables names, you can resolve the conversation if it is all good